tests/fastapi/test_loggin_route.py

```
@@ -0,0 +1,339 @@
+def test_logger_internal_server_error(
+    fastapi_client: TestClient, caplog
+) -> None:
+    """
+    Test that verifies:
+    - A request that causes a 500 error is logged.
+    """
+    caplog.set_level(logging.INFO)
+
+    request_data = {'some_field': 'some value'}
+
+    with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)